roda-ui/roda-wui/src/main/java/org/roda/wui/client/common/dialogs/Dialogs.java

```diff
@@ -112,6 +121,69 @@ public static void showInformationDialog(String title, final String message, Str
     showInformationDialog(title, message, continueButtonText, canCopyMessage, new NoAsyncCallback<>());
   }
 
+  public static void showTechnicalMetadataInformation(String title, HashMap<String, String> tikaValues,
```

Comment:
Do you know if this method is still used? Shouldn't a stylesheet provide the HTML file to be displayed?